docs/source/tracking.rst

```diff
@@ -272,8 +272,9 @@ specified as ``db_type://<user_name>:<password>@<host>:<port>/<database_name>``.
 database types are ``mysql``, ``mssql``, ``sqlite``, and ``postgresql``.
 
 By default ``--backend-store-uri`` is set to the local ``./mlruns`` directory (the same as when
-running ``mlflow run`` locally), but when running a server, make sure that this points to a
-persistent (that is, non-ephemeral) file system location.
+running ``mlflow run`` locally). But when running a server, make sure that this points to a
+location on a persistent (that is, non-ephemeral) file system. If the location already exists,
+it must be a valid backend store, otherwise it will be created automatically.
```

Comment:
This change is making the documentation unclear. What do you mean "If the location exists"? Depending on the type of backend store, this could mean different things. Also. not clear what it means by "will be created automatically". Is this specific for file-backed store or DB store. The action being done would be different. If you want to document specific actions done by these, that would be great. Otherwise, this new language is more confusing.

This same confusion in the comments below in CLI commands below...

Reply:
Thanks for the comments. I have only tried file-backed store, so I might have missed something. This change is mainly inspired by #989. There I tried two scenarios: passing `--file-store` a local path as well as a URI pointing to a path on the tracking server. If the path (local or on the server) is an existent empty directory, MLflow would complain "Could not find experiment with ID 0". And the cause is: MLflow expects the directory to be either nonexistent, in which case it will be automatically created, or a valid file store directory.  This behavior is not documented in the docs or help, neither does the error message indicate this. Both @GuillemGSubies and I think this behavior needs to be documented, and hence this change. Feel free to suggest a better solution.